sm/safe_math_util.h

```diff
@@ -0,0 +1,12 @@
+#ifndef _SAFE_MATH_UTIL_H_
+#define _SAFE_MATH_UTIL_H_
+
+// TODO: We may want to add checks for GCC/Clang version. We also want
+// to add wrappers for other checked math functions.
+
+#define __checked_uadd(a, b, out)   (__builtin_uadd_overflow(a, b, out))
```

Comment:
what are these for?

Reply:
These were added for the wrapped encl_bitmap checks, before it was removed. We will just want safe math functions available in general. Not used  right now but not a bad idea to have them.